resources/assets/sass/ui.scss

```diff
@@ -154,10 +154,11 @@ legend {
   width: 125px;
 }
 
-@media(min-width: 400px){
-  height: 175px;
-  width: 175px;
-}
+// TODO: What element should this be for?
```

Comment:
@queryluke This media query broke the compiler because it wasn't referring to a css element - do you know what it should be referring to, or should I just remove this?

Reply:
oh, that's odd. well if it's not referencing a css element, then it's not doing anything. so yeah I'd call it safe to delete.

Reply:
Deleted.